src/course-unit/data/thunk.js

```diff
@@ -110,6 +121,31 @@ export function editCourseItemQuery(itemId, displayName, sequenceId) {
   };
 }
 
+export function editCourseUnitVisibilityAndData(itemId, type, isVisible) {
+  return async (dispatch) => {
+    dispatch(updateSavingStatus({ status: RequestStatus.PENDING }));
+    dispatch(updateQueryPendingStatus(true));
+    const notificationMessage = getNotificationMessage(type, isVisible);
+    dispatch(showProcessingNotification(notificationMessage));
```

Comment:
Are the processing notifications internationalized?

Reply:
No, we will add internationalization later as a separate PR